### PR TITLE
Add basic tests for electric toolbox

### PR DESCRIPTION
**Pull request by Trophime**
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/feelpp/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes?
- [ ] Have you successfully run the Feel++ testsuite with your changes locally?
- [ ] Have you written Doxygen comments in your contribution ?

-----

### CODE REVIEW

File: toolboxes/electric/busbar/2d.cfg [hunk 1]
```diff
@@ -0,0 +1,30 @@
+fe-approximation=P1 #P1,P2
+directory=toolboxes/thermoelectric/cvg2d
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.cfg]: it's not  thermoelectric app but electric

File: toolboxes/electric/busbar/2d.cfg [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+[thermo-electric]
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.cfg]: it's not the good prefix

File: toolboxes/electric/busbar/2d.cfg [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+mesh.filename=$cfgdir/2d.geo
+filename=$cfgdir/2d.json
+
+[gmsh]
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.cfg]: it's not the good prefix, it's [electric.gmsh]

File: toolboxes/electric/busbar/2d.cfg [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+hsize=0.1
+partition=1
+
+[cvg]
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.cfg]: not valid option

File: toolboxes/electric/busbar/2d.cfg [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+nb-iter=5
+factor=2
+filename=cvg.dat
+
+#[thermo-electric.gmsh]
+#scale=1000 # from mm to m
+
+[thermo-electric.heat]
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.cfg]: no heat

File: toolboxes/electric/busbar/2d.cfg [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+#initial-solution.temperature=293
+#verbose=1
+verbose_solvertimer=1
+pc-type=gamg
+ksp-monitor=1
+
+[thermo-electric.electric]
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.cfg]: it's electric

File: toolboxes/electric/busbar/2d.json [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                "V1":
+                {
+                    "expr":"0.25" // V_out
+                }
+            }
+        }
+    },
+    "PostProcess":
+    {
+        "use-model-name":1,
```
Comment (vincentchabannes) [inline toolboxes/electric/busbar/2d.json]: no need use-model-name" with monophysic toolbox

File: toolboxes/electric/quarter-turn/2d.json [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    },
+    "PostProcess":
+    {
+        "use-model-name":1,
+        "electric":
+        {
+            "Exports":
+            {
+                "fields":["electric.electric-potential","electric.electric-field","electric.pid"]
```
Comment (vincentchabannes) [inline toolboxes/electric/quarter-turn/2d.json]: no electric prefix

Comment (Trophime) [reply toolboxes/electric/quarter-turn/2d.json]: @vincentchabannes I've made the changes you request.
I still need to run the tests to  be 100% sure of the changes.

**Comment by prudhomm** [comment]
where do we stand? can it be merged?

**Comment by prudhomm** [comment]
@vincentchabannes ?

**Pull request merged**